### PR TITLE
Ins 2008 machines manager component

### PR DESCRIPTION
**Pull request by ruz**

### CODE REVIEW

File: logicrunner/logicrunner_unit_test.go [hunk 1]
```diff
@@ -78,38 +79,31 @@ func (suite *LogicRunnerCommonTestSuite) BeforeTest(suiteName, testName string)
...
-	suite.mc.Finish()
-
-	for _, e := range suite.lr.Executors {
-		if e == nil {
-			continue
-		}
-		// e.Stop() is about to be called in LogicRunner.Stop() method
-		e.(*testutils.MachineLogicExecutorMock).StopMock.Expect().Return(nil)
-	}
+	//suite.mc.Wait(2 * time.Minute)
```
Comment (bigbes) [inline logicrunner/logicrunner_unit_test.go]: commeneted?

File: version/manager/versiontable.go [hunk 2]
```diff
@@ -4,4 +4,4 @@ package manager
 
 func (vm *VersionManager) loadVersionTable() {
 	return
-}
+}
```
Comment (bigbes) [inline version/manager/versiontable.go]: newline is absent

File: logicrunner/machinesmanager.go [hunk 3]
```diff
@@ -0,0 +1,39 @@
+
```
Comment (bigbes) [inline logicrunner/machinesmanager.go]: license is absent

File: logicrunner/logicrunner.go [hunk 4]
```diff
@@ -149,6 +149,8 @@ func NewLogicRunner(cfg *configuration.LogicRunner) (*LogicRunner, error) {
 	return &res, nil
 }
 
+func (lr *LogicRunner) LRI() {}
```
Comment (bigbes) [inline logicrunner/logicrunner.go]: What's purpose of LRI? To distinguish LR and OS/ES?

Comment (ruz) [reply logicrunner/logicrunner.go]: To distinguish LogicRunner in out DI from other classes that have OnPulse

File: logicrunner/logicrunner_unit_test.go (hunk 1 above)
Comment (ruz) [reply logicrunner/logicrunner_unit_test.go]: fixing

File: logicrunner/machinesmanager.go (hunk 3 above)
Comment (ruz) [reply logicrunner/machinesmanager.go]: fixed

File: version/manager/versiontable.go (hunk 2 above)
Comment (ruz) [reply version/manager/versiontable.go]: fixed

**Pull request merged**